### PR TITLE
doc: Registering calico-windows-v3.17.1

### PR DESCRIPTION
**Pull request by AbelHu**
- Calico Windows is from https://docs.projectcalico.org/release-notes/
- calico-windows-v3.17.1.zip is signed by Microsoft aks-engine team
- We use `./vhdbuilder/cgmanifest.json` to register this third party package.

### CODE REVIEW

File: vhdbuilder/cgmanifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "Registrations": [
+        {
+            "Component": {
+                "Type": "other",
+                "Other": {
+                    "Name": "CalicoWindows",
```
Comment (mainred) [inline vhdbuilder/cgmanifest.json:7]: 1. how should we relate the `CalicoWindows` to the calico component we are using?
2. how can we verify its correctness?  is this tool useful? https://docs.opensource.microsoft.com/tools/cg/cgmanifest.html#how-can-i-test-my-cgmanifestjson-file cgmanifest.json

Comment (AbelHu) [reply vhdbuilder/cgmanifest.json:7]: For #1, can you add more details for your question? From my understanding, it is just a name. The source URL is more important.
For #2, when creating this PR with this file cgmanifest.json, the reviewer `microsoftopensource` is automatically recommended so I think that it should be good. And I will wait the review from Microsoft opensource team. It seems like building the tool needs more time. We can do it later if it worths.

**Comment by AbelHu** [comment]
ping @microsoftopensource

File: vhdbuilder/cgmanifest.json (hunk 1 above)
Comment (jcfiorenzano) [reply vhdbuilder/cgmanifest.json:7]: Hi I work in the Open Source Engineering team

1. In the case of "other" components as @AbelHu mention the most important part is the URL, think about the name as the display name.
2. The cgmanifest looks good to me, but you can use to tool to validate that the file is going to be parsed successfully by our detectors.

Comment (AbelHu) [reply vhdbuilder/cgmanifest.json:7]: @jcfiorenzano Thanks for your review. We will follow the guidance to merge the PR and then check the alerts.

Comment (mainred) [reply vhdbuilder/cgmanifest.json:7]: @jcfiorenzano may I ask do we need to update the version and DownloadUrl every time we have an update of this component? thanks

**Pull request merged**